llama.h

```diff
@@ -195,6 +195,7 @@ extern "C" {
         bool vocab_only; // only load the vocabulary, no weights
         bool use_mmap;   // use mmap if possible
         bool use_mlock;  // force system to keep model in RAM
+        bool use_awq;    // whether to use awq quantization
```

Comment:
I think an AWQ model should use the activation scales by default. AFAIU using an AWQ model with applying the activation scales would produce incorrect results because the FFN data has been scaled, so I don't see a reason to be able to turn off AWQ